src/components/MobileMenu.tsx

```diff
@@ -42,14 +45,27 @@ const MobileMenu: React.VFC<MobileMenuProps> = (props) => {
                 <CloseIcon />
               </button>
             </div>
-            <h2 className="list-header">Navigation</h2>
             <ul>
               <li>
-                <a href="/operators">Operators</a>
-              </li>
-              <li>
-                <a href="/about">About</a>
+                <div className="search-bar-container">
+                  <SearchBar
+                    placeholder="Search"
+                    onInputChange={(input) => {
+                      setSearchOpen(!!input);
+                    }}
+                  />
+                </div>
               </li>
+              {!isSearchOpen && (
+                <li>
+                  <a href="/operators">Operators</a>
+                </li>
+              )}
+              {!isSearchOpen && (
+                <li>
+                  <a href="/about">About</a>
+                </li>
+              )}
```

Comment:
The duplication of the boolean here can be avoided by wrapping both in a Fragment:

```ts
{!isSearchOpen && (
  <Fragment>
    <li>
      ...
    </li>
    <li>
      ...
    </li>
  </Fragment>
)}
```